packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -75,7 +73,7 @@ See [below](#how-grouped-batching-works) for an example.
 
 ### Risks
 
-This option is experimental and should not be enabled yet in production. This option should **ONLY** be enabled after observing that 99.9% of your application sessions contains these changes (runtime version "2.0.0-internal.4.1.0" or later). Containers created with this option may not open in future versions of the framework.
+This option should **ONLY** be enabled after observing that 99.9% of your application sessions contains these changes (runtime version "2.0.0-internal.4.1.0" or later). Containers created with this option may not open in future versions of the framework.
```

Comment:
```suggestion
This option should **ONLY** be enabled after observing that 99.9% of your application sessions contains these changes (runtime version "2.0.0-internal.7.0.0" or later). Containers created with this option may not open in future versions of the framework.
```
I think the version needs to be updated as well from the assert fixes a couple months ago.